tactics/class_tactics.ml

```diff
@@ -100,7 +100,7 @@ let _ =
 
 let _ =
   declare_bool_option
-    { optdepr  = false;
+    { optdepr  = true; (* remove in 8.8 *)
```

Comment:
OK @mattam82 confirmed off-line that this is not really a compatibility option (in the sense that it is unclear when this option can be turned off by default, not in 8.7 at least). The documentation needs updating.